internal/tensorzero-node/lib/bindings/OrderDirection.ts

```diff
@@ -1,3 +1,6 @@
 // This file was generated by [ts-rs](https://github.com/Aleph-Alpha/ts-rs). Do not edit this file manually.
 
+/**
+ * Order by terms for querying inferences.
+ */
 export type OrderDirection = "ASC" | "DESC";
```

Comment:
This feels inconsistent with other APIs, we might as well use "ascending" and "descending" since we don't explicitly use SQL-like keywords anywhere else.